### PR TITLE
Add Audio Connector API

### PR DESCRIPTION
**Pull request by maxkahan**
This adds Audio Connector API functionality into the SDK, and readies it for a minor release.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Release v3.4.0
+
+- Support for Audio Streamer (lite) API via `stream_audio_to_websocket` method
```
Comment (jeffswartz) [inline CHANGES.md:3]: Please change "Audio Streamer (lite)" to "Audio Streamer" throughout.

File: README.rst [hunk 2]
```diff
@@ -551,6 +551,33 @@ For more information about OpenTok live streaming broadcasts, see the
 `Broadcast developer guide <https://tokbox.com/developer/guides/broadcast/>`_.
 
 
+Streaming audio to a websocket
```
Comment (jeffswartz) [inline README.rst:554]: Please change "websocket" to "WebSocket" in the docs throughout. (But do not change the names in the API, such as `stream_audio_to_websocket`.)

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (maxkahan) [reply CHANGES.md:3]: As discussed over slack, will rename to "Audio Connector"

File: README.rst (hunk 2 above)
Comment (maxkahan) [reply README.rst]: Have done this now. Note I changed the method name `stream_audio_to_websocket` to `connect_audio_to_websocket` as that fits the new product name better.

File: CHANGES.md (hunk 1 above)
Comment (maxkahan) [reply CHANGES.md]: Have renamed throughout as requested

**Review by jeffswartz — APPROVED** [review]

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**